src/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+include(GNUInstallDirs)
```

Comment:
@climbfuji I added this to the host Cmake for the SCM to make this available to physics and framework. If the host doesn't set this, will you also need to also add it to the [physics](https://github.com/ufs-community/ccpp-physics/pull/259/files)?

Reply:
I added it for both.

Reply:
d5776deebd884ec2c6e179094dd2b575f628f6e1

Reply:
Of course you did!